src/ray/object_manager/plasma/store.cc

```diff
@@ -805,10 +621,7 @@ Status PlasmaStore::ProcessMessage(const std::shared_ptr<Client> &client,
     // This code path should only be used for testing.
     int64_t num_bytes;
     RAY_RETURN_NOT_OK(ReadEvictRequest(input, input_size, &num_bytes));
-    std::vector<ObjectID> objects_to_evict;
-    int64_t num_bytes_evicted =
-        eviction_policy_.ChooseObjectsToEvict(num_bytes, &objects_to_evict);
-    EvictObjects(objects_to_evict);
+    int64_t num_bytes_evicted = object_lifecycle_mgr_.RequireSpace(num_bytes);
```

Comment:
Actually, I don't think we are using this Evict call. Can you verify that? I think we can delete it and remove `RequireSpace ` from the public interface